prql-compiler/src/test.rs

```diff
@@ -2954,22 +2954,20 @@ fn test_errors() {
        ╭─[:1:23]
        │
      1 │ Mississippi has four S’s and four I’s.
-       ·                       ┬
-       ·                       ╰── unexpected ’
+       │                       ┬
+       │                       ╰── unexpected ’
     ───╯
     Error:
        ╭─[:1:36]
        │
      1 │ Mississippi has four S’s and four I’s.
-       ·                                    ┬
-       ·                                    ╰── unexpected ’
+       │                                    ┬
+       │                                    ╰── unexpected ’
     ───╯
     Error:
        ╭─[:1:39]
        │
-     1 │ Mississippi has four S’s and four I’s.
-       ·                                       ┬
-       ·                                       ╰── Expected * or an identifier, but didn't find anything before the end.
```

Comment:
The current problem is that we're losing these lines. I had a look but couldn't see why. If anyone wants to take a look, great, otherwise I'll look more